### DISCUSSION TITLE
New version ETA?

### DISCUSSION THREAD

**Discussion by rdykiel** [victools/jsonschema-generator#493] [General]
Hi I would need this feature https://github.com/victools/jsonschema-generator/pull/492
Can you let me know what's the ETA of the new version?
Thanks

**Comment by CarstenWickner** [comment]
Hi @rdykiel,

I've just published release [v4.37.0](https://github.com/victools/jsonschema-generator/releases/tag/v4.37.0).
It may take a couple of hours until it is all propagated through the Maven repositories.

**Reply by rdykiel** [reply]
Thanks!
